docs/sources/administration/organization-management/index.md

```diff
@@ -8,6 +8,9 @@ description: Describes how to use organizations to isolate dashboard to users an
 keywords:
   - organizations
   - dashboards
+labels:
+  products:
+    - oss
```

Comment:
This is also Enterprise. In the Grafana repo, all is OSS and enterprise except for the enterprise-only topics (there is a Note in each enterprise only topic mentioning that it is enterprise only).